incubator/consul/templates/consul-petset.yaml

```diff
@@ -157,6 +157,12 @@ spec:
       - name: gossip-key
         secret:
           secretName: gossip-key
+      readinessProbe:
+        httpGet:
+          path: /v1/health/service/consul
```

Comment:
@chrislovecnm - It should work and always return details of the servers in the cluster. I haven't had an issue where it hasn't worked using this chart. Have you? Is there another endpoint that would better server as readinessProbe?